tests/test_scan_and_result.py

```diff
@@ -229,6 +231,45 @@ def test_get_default_help(self):
         self.assertEqual(response.get('status'), '200')
         self.assertEqual(response.tag, 'help_response')
 
+    @patch('ospd.ospd.subprocess')
+    def test_get_performance(self, mock_subproc):
+        daemon = DummyWrapper([])
+        mock_subproc.check_output.return_value("foo")
+        response = secET.fromstring(
+            daemon.handle_command(
+                '<get_performance start="0" end="0" titles="mem"/>')
+        )
+
+        self.assertEqual(response.get('status'), '200')
+        self.assertEqual(response.tag, 'get_performance_response')
+
+    def test_get_performance_fail_int(self):
+        daemon = DummyWrapper([])
+        cmd = secET.fromstring(
+            '<get_performance start="a" end="0" titles="mem"/>')
+
+        self.assertRaises(
```

Comment:
Personally I would use the with statement for assert raises (see https://docs.python.org/3/library/unittest.html#unittest.TestCase.assertRaises) but it's up to you :-)

Reply:
Other tests don't use the with statement for assert raises neither. I will do this in a future PR.